### ISSUE TITLE
Accessibility: Fix MSSQL query editor fastpass issues

### ISSUE THREAD

**Issue by kaydelaney** [grafana/grafana#42716]

**Comment by codeincarnate** [comment]
Closing as this should be fixed by #43981
